coderd/healthcheck/derphealth/derp.go

```diff
@@ -107,9 +109,26 @@ func (r *Report) Run(ctx context.Context, opts *ReportOptions) {
 	ncReport, netcheckErr := nc.GetReport(ctx, opts.DERPMap)
 	r.Netcheck = ncReport
 	r.NetcheckErr = convertError(netcheckErr)
+	if mapVaryDest, _ := r.Netcheck.MappingVariesByDestIP.Get(); mapVaryDest {
+		r.Warnings = append(r.Warnings, health.Messagef(health.CodeSTUNMapVaryDest, stunMapVaryDest))
+	}
 
 	wg.Wait()
 
+	// Count the number of STUN-capable nodes.
+	var stunCapableNodes int
+	for _, region := range r.Regions {
+		for _, node := range region.NodeReports {
+			if node.STUN.CanSTUN {
+				stunCapableNodes++
+			}
+		}
+	}
+	if stunCapableNodes == 0 {
+		r.Severity = health.SeverityWarning
```

Comment:
Does it still warn if STUN handling is disabled?

Reply:
Good point, this should now not alert if there are no STUN servers configured.